app/src/main/java/com/orangeelephant/sobriety/ui/screens/create/Create.kt

```diff
@@ -0,0 +1,224 @@
+        ) {
+
+            // photo
+            Column(
+                verticalArrangement = Arrangement.Top,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(16.dp)
+            ) {
+                Spacer(modifier = Modifier.height(24.dp))
+                CircleImagePicker(
+                    selectedImageUri = createScreenViewModel.selectedImageUri,
+                    onImageSelected = { uri -> createScreenViewModel.selectedImageUri.value = uri }
+                )
+            }
+
+            // user input
+            Column (
+                verticalArrangement = Arrangement.Bottom,
+                horizontalAlignment = Alignment.CenterHorizontally,
+                modifier = Modifier
+                    .padding(bottom = 100.dp, top = 40.dp, start = 40.dp, end = 40.dp )
+                    .verticalScroll(rememberScrollState()) // Enable vertical scrolling
+
+            ) {
+
+                // name
+                OutlinedTextField(
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    value = createScreenViewModel.nameText,
+                    onValueChange = { createScreenViewModel.nameText = it },
+                    label = { Text(stringResource(R.string.placeholder_counter_name)) }
+                )
+
+                Spacer(modifier = Modifier.height(8.dp))
+
+                // date
+                ClickableOutlinedTextField(
+                    modifier = Modifier.fillMaxWidth(),
+                    value =  TextFieldValue(convertMillisecondsToDate(createScreenViewModel.dateVal)),
+                    onClick = { isDialogOpen = true },
+                    label = { Text(context.getString(R.string.placeholder_date)) }
+                )
+
+                Spacer(modifier = Modifier.height(8.dp))
+
+                // reasons
+                if (numberOfTextFields > textValues.size) {
+                    repeat(numberOfTextFields - textValues.size) {
+                        textValues.add(TextFieldValue())
+                    }
+                }
+
+                for (i in 0 until numberOfTextFields) {
+                    OutlinedTextField(
+                        modifier = Modifier.fillMaxWidth(),
+                        value = textValues[i],
+                        onValueChange = { textValues[i] = it },
+                        label = { Text(stringResource(R.string.placeholder_counter_reason, i + 1)) }
+
+                    )
+                }
+
+                if (textValues[0] != TextFieldValue("")) {
+                    Button(
+                        onClick = { numberOfTextFields++ }
+                    ) {
+                        Text(text = context.getString(R.string.add_additional_reason))
+                    }
+                }
+            }
+        }
+    }
+}
+
+// create compostable preview
```

Comment:
Are you actually leaving this as compostable

Reply:
...no